frontend/public/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -110,6 +110,10 @@ export const OperatorHubSubscribeForm = withFormState((props: OperatorHubSubscri
       metadata: {
         name: packageName,
         namespace: props.formState().targetNamespace,
+        labels: {
```

Comment:
Use these labels to determine if we need to update/delete a `CatalogSourceConfig` when deleting `Subscription`.